versions.gradle

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2025, The casual project. All rights reserved.
+ *
+ * This software is licensed under the MIT license, https://opensource.org/licenses/MIT
+ */
+
+// casual-java group and version
+group = 'se.laz.casual'
+version = '0.0.1-beta'
+
+ext.libs = [
+  fabric8: 'io.fabric8:kubernetes-client:7.1.0',
```

Comment:
There's a 7.3.1 version available

Reply:
Thanks, they came out whilst I was building, this was the latest and greatest when i started :)
Bumped to 7.3.1.